extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -609,8 +606,18 @@ export class AmpA4A extends AMP.BaseElement {
           // Load any extensions; do not wait on their promises as this
           // is just to prefetch.
           const extensions = extensionsFor(this.win);
-          creativeMetaDataDef.customElementExtensions.forEach(
-            extensionId => extensions.loadExtension(extensionId));
+          creativeMetaDataDef.extensions.forEach(
+            extension => {
+              const extensionId = extension['custom-element'];
+              const extensionSrc = extension['src'];
+              const versionStartIndex = extensionSrc.indexOf(extensionId) +
+                  extensionId.length + 1;
+              // -3 because the src string always ends with '.js'.
+              const versionEndIndex = extensionSrc.length - 3;
+              const extensionVer = extensionSrc.substring(
+                  versionStartIndex, versionEndIndex);
+              extensions.loadExtension(extensionId, extensionVer);
```

Comment:
Not sure what you mean. The extensions metadata is formatted like:

```
"customElementExtensions" : [
   {
      "custom-element": "amp-analytics",
      "src": "https://cdn.ampproject.org/v0/amp-analytics-0.1.js"
   },
   {
      "custom-element": "amp-list",
      "src": "https://cdn.ampproject.org/v0/amp-list-0.1.js"
   },
]
```

I pull the version number from the src string. So long as you specify the correct version on the src string, this should work.